kokoro/linux/64-bit/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN echo "deb http://ppa.launchpad.net/ondrej/php/ubuntu trusty main" | tee /etc
 # Install dotnet SDK based on https://www.microsoft.com/net/core#debian
 # (Ubuntu instructions need apt to support https)
 RUN apt-get update && apt-get install -y --force-yes curl libunwind8 gettext && \
-  curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?LinkID=847105 &&  \
+  curl -sSL -o dotnet.tar.gz https://download.visualstudio.microsoft.com/download/pr/69937b49-a877-4ced-81e6-286620b390ab/8ab938cf6f5e83b2221630354160ef21/dotnet-sdk-2.2.104-linux-x64.tar.gz &&  \
```

Comment:
looks like other projects are still using `netcoreapp1.0`, so they will need the older SDK installed to be able to run.

Reply:
Good point. We should work out whether we want to still test against netcoreapp1.0 (by installing multiple SDKs), or update *everything* to 2.x at the same time.

Reply:
I think we should update all the .exe projects to netcoreapp2.1  because that corresponds to an LTS version of the SDK (btw, that strategy is not based on a MSFT best practice or anything, it just seems as logical choice to me after experiencing some of the pains with missing SDK versions and limitations of global.json that doesn't allow wildcards)

Reply:
That SGTM. How about I split that task into a separate PR that *just* updates stuff, and we can get that working before coming back to benchmarks?

Reply:
That would be great. Btw I think I have the ability to rebuild and push the dockerimage once needed.

Reply:
Btw we also need to be careful about not breaking https://github.com/protocolbuffers/protobuf/blob/8bdecd743061317e4377f632db352ade797a1b8e/csharp/build_packages.bat#L4

Reply:
Right. So we need to update the Windows image as well. Are you in a position to do that?

Reply:
There's no simple way to "update windows image" on kokoro. There's a version of dotnet SDK preinstalled, but I don't know what exactly it is. For windows build, we might need to install dotnet SDK via a script. I'm actually surprised that we don't have a presubmit build for protobuf C# on Windows - I wil try to work with protobuf team to setup one.

Reply:
There's a PR already up for updating projects over at #5838

Reply:
@ObsidianMinor: Wonderful, thanks very much :)